addon/components/new-osf-navbar/template.hbs

```diff
@@ -46,7 +46,20 @@
                             </li>
                         {{else}}
                             {{!SERVICE LINK}}
-                            <li><a href="{{navLink.href}}"> {{navLink.name}}</a></li>
+                            <li>
+                                <a href="{{navLink.href}}">
+                                    {{#if (eq navLink.type 'addAPreprint')}}
+                                        <span class="hidden-xs hidden-sm">
+                                            {{navLink.name}}
+                                        </span>
+                                        <span class="hidden-md hidden-lg hidden-xl">
```

Comment:
Yeah, I'm pretty sure those are new in v4. I think we just have http://getbootstrap.com/css/#responsive-utilities-classes available to us currently.